java/read-geopackage/README.md

```diff
@@ -1,25 +1,32 @@
 # Read GeoPackage
-Read rasters and feature tables from geopackages and display them as layers in a map.
 
-![Read GeoPackage App](read-geopackage.png)
+Add rasters and feature tables from a GeoPackage to a map.
+
+![Image of read GeoPackage](read-geopackage.png)
+
+## Use case
+
+The OGC GeoPackage specification defines an open standard for sharing raster and vector data. You may want to use GeoPackage files to support file-based sharing of geographic data.
 
 ## How to use the sample
-The `Layer`s in the `GeoPackage` are shown in a drawer on the left hand side of the screen. Click on a `Layer` to add or remove it from map.
+
+When the sample loads, the feature tables and rasters from the GeoPackage will be shown on the map.
```

Comment:
It looks like there is a menu on the side here to select layers to add, perhaps this should be mentioned here?